server/service/orbit_client_test.go

```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestGetConfig(t *testing.T) {
+	t.Run(
+		"config cache", func(t *testing.T) {
+			oc := OrbitClient{}
+			oc.configCache.config = &fleet.OrbitConfig{}
+			oc.configCache.lastUpdated = time.Now().Add(1 * time.Second)
+			config, err := oc.GetConfig()
+			require.NoError(t, err)
+			require.Equal(t, oc.configCache.config, config)
+		},
```

Comment:
should we add a test for the error case as well?

Reply:
Yes, added.